### PR TITLE
Aragon Monthly #5 Submission 

### PR DESCRIPTION
**Pull request by pet3r-pan**
# New content for Aragon Monthly

- I left the featured section blank for editors to add / edit
- I couldn't find any articles and classified so I left them empty
- I kept the descriptions of the news items between 45-60 words

+ Also ps. I am kinda of confused in how this status bounty thing works? What does a bounty process look like? I signed up and logged in using github. Kinda confused

### CODE REVIEW

**Comment by john-light** [comment]
Hi @pet3r-pan thank you for your Pull Request! Regarding your question about Status Open Bounty, we have published a guide for using the app to claim bounties here: http://wiki.aragon.one/tutorials/Status_Open_Bounty_Guide/

File: docs/news/index.md [hunk 1]
```diff
@@ -1,28 +1,59 @@
...
-| | |  
-
-| | |  
-:-----------:|:-----------:|:-----------:|  
-| | |  
-| | |  
-| | |  
-| | |  
-| | |  
+|[Should Couchsurfing Be a DAO?](https://hackernoon.com/should-couchsurfing-be-a-dao-6507646e34ef)|[Curate This: Token Curated Registries That Don’t Work](https://blog.coinfund.io/curate-this-token-curated-registries-that-dont-work-d76370b77150)| [‘Hundreds’ Of DAOs Running on Aragon Says CTO In Exclusive Interview](https://bitsonline.com/aragon-daos-interview/)|
```
Comment (Smokyish) [inline docs/news/index.md]: All `Author` links should point to the profile in their respective publication, for example, `Chelsea Rustrum` link should point to `https://hackernoon.com/@shareablelife`.

Please check and fix the `Author` links on each news item and also on `tag` pages.

File: docs/news/daos.md [hunk 2]
```diff
@@ -1,28 +1,11 @@
...
-| | |  
-
-| | |  
-:-----------:|:-----------:|:-----------:|  
-| | |  
-| | |  
-| | |  
-| | |  
-| | |  
+|[Should Couchsurfing Be a DAO?](https://hackernoon.com/should-couchsurfing-be-a-dao-6507646e34ef)|[Never Heard of a DAO? Here's Why It's Going to Become The Hottest New Type of Corporation](https://www.inc.com/darren-heitner/never-heard-of-a-dao-heres-why-its-going-to-become-hottest-new-type-of-corporation.html)|
```
Comment (Smokyish) [inline docs/news/daos.md]: Table is missing the third, empty, column.

File: docs/news/aragon.md [hunk 3]
```diff
@@ -1,28 +1,35 @@
...
+|[Read More](https://blog.aragon.one/introducing-aragon-one-b14dd804c5ce)|[Read More](https://blog.aragon.one/decentralizing-aragons-development-iii-onboarding-new-teams-32786cb805a5)|[Read More](https://sludgefeed.com/aragon-releases-new-developer-portal-for-building-dapps-and-daos/)|
+
+|[Releasing the Aragon Developer Portal](https://blog.aragon.one/releasing-the-aragon-developer-portal-237a8dbc8a48)|[Aragon Q1 Transparency Report](https://blog.aragon.one/aragon-q1-transparency-report-5ba9fc435ca6)|
```
Comment (Smokyish) [inline docs/news/aragon.md]: Table is missing the third, empty, column.

File: docs/news/index.md [hunk 4]
```diff
@@ -1,28 +1,59 @@
...
+|[Read More](https://sludgefeed.com/aragon-releases-new-developer-portal-for-building-dapps-and-daos/)|[Read More](https://blog.aragon.one/releasing-the-aragon-developer-portal-237a8dbc8a48)|[Read More](https://blog.aragon.one/aragon-q1-transparency-report-5ba9fc435ca6)|"
+
+|[We Need to Formalize Governance Research using DAOs](https://medium.com/betoken/we-need-to-formalize-governance-research-using-daos-e9393da427f3)|
```
[the remark this answers is not in the document]
Comment (pet3r-pan) [reply docs/news/index.md]: Not sure what you mean by link but I understood everything else. I will leave the columns empty like how other editions have done.

File: docs/news/index.md [hunk 5]
```diff
@@ -1,28 +1,59 @@
...
+|[Read More](https://blog.aragon.one/the-aragon-manifesto-4a21212eac03)|[Read More](https://blog.aragon.one/aragon-nest-first-awarded-grants-29a2f3aa8d4a)|[Read More](https://blog.aragon.one/aragon-q1-development-update-b5525e9477c4)|
+
+|[First community-wide Aragon Feedback Survey!](https://blog.aragon.one/first-community-wide-aragon-feedback-survey-132df9547fbc)|[Introducing Aragon One](https://blog.aragon.one/introducing-aragon-one-b14dd804c5ce)| [Decentralizing Aragon’s development III: Onboarding new teams](https://blog.aragon.one/decentralizing-aragons-development-iii-onboarding-new-teams-32786cb805a5)|
```
[the remark this answers is not in the document]
Comment (pet3r-pan) [reply docs/news/index.md]: Done! I deviated from the main image in hopes to adding variety to the thumbnails.

File: docs/news/daos.md (hunk 2 above)
Comment (pet3r-pan) [reply docs/news/daos.md]: Done

File: docs/news/aragon.md (hunk 3 above)
Comment (pet3r-pan) [reply docs/news/aragon.md]: Done

**Comment by Smokyish** [comment]
@pet3r-pan Please make sure you have your details covered as explained in the [How to claim an Aragon bounty using Status Open Bounty](https://wiki.aragon.one/tutorials/Status_Open_Bounty_Guide/) tutorial, your Pull Request has been merged and a final payout will be confirmed within a week.

**Comment by Smokyish** [comment]
@pet3r-pan What I just realized after merging the Pull Request is that it was missing the `Closes #141` from the PR description, and that would mean you couldn't claim the bounty. 

Please submit a new PR from your `pet3r-pan:master` master branch that should have the recent changes from this one with so we can get the bounty properly rewarded to you!

**Comment by john-light** [comment]
@Smokyish it's not only the bounty for the Editor, but also for all the individual issues that had bounties (if there were any). How can we make sure @pet3r-pan gets those bounties?

**Comment by Smokyish** [comment]
@john-light There were no active bounties for Issue 05.

**Comment by pet3r-pan** [comment]
@john-light Damn. My bad. I knew I was missing something.

@Smokyish I saw a bounty on here: https://openbounty.status.im/app#/?owner=aragon and thus led me to this issue. Let me know if I misunderstood anything

I will re-submit a PR with the reference.

**Pull request merged**